lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -1,6 +1,21 @@
 <?xml version="1.0"?>
 <datatypes>
   <registration converters_path="lib/galaxy/datatypes/converters" display_path="display_applications">
+
+    <!-- Various source code files -->
+    <datatype extension="source.h" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true" description="C or cpp header file"/>
+    <datatype extension="source.C" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true" description="C source file"/>
+    <datatype extension="source.cpp" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true" description="C++ source file" />
+    <datatype extension="source.py" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true" description="Python source file"/>
+    <datatype extension="source.go" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true" description="Go source file"/>
+    <!-- End various source code files -->
+
+    <!-- ROOT statistical analysis binary file - Used typicall in the High Energy Physics community -->
+    <datatype extension="root" type="galaxy.datatypes.binary:Binary" subclass="true" display_in_upload="true" description="ROOT binary file."/>
```

Comment:
Can we use a more specific name/extension here? `root` seems to be too generic.

Maybe `hep.root`?

Reply:
sure! Was thinking that this could be nice.